extractor/filesystem/containers/k8simage/testdata/StatefulSet.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: apps/v1
```

Comment:
Can we reduce most of the testfiles to be minimal? I.e., only contain those values and properties that are absolutely necessary? 

This greatly improves test readability and maintainability. We can still have on test with bloated properties to make sure that the parser does not error out on real-world files, however, for most testcases we should favor minimal testcases to reduce reviewing and readability bloat.